### PR TITLE
postgresql_info return values doc section: leave only top level keys info

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Fixes https://github.com/ansible-collections/community.postgresql/issues/754

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review

**Comment by Andersson007** [comment]
ready for review

File: plugins/modules/postgresql_info.py [hunk 1]
```diff
@@ -151,378 +121,36 @@
...
-    valid_until:
-      description:
-      - Password expiration date U(https://www.postgresql.org/docs/current/sql-alterrole.html).
-      returned: success
-      type: str
-      sample: "9999-12-31T23:59:59.999999+00:00"
 pending_restart_settings:
   description:
-  - List of settings that are pending restart to be set.
+  - List of settings that are pending until restart to be set.
```
Comment (hunleyd) [inline plugins/modules/postgresql_info.py:146]:
```suggestion
  - List of changed settings that will take effect upon restart .
```

how's that?

**Review by hunleyd — CHANGES_REQUESTED** [review]

**Comment by Andersson007** [comment]
@hunleyd your suggestion LGTM, committed, PTAL

**Review by hunleyd — APPROVED** [review]

**Pull request merged**